frontend/src/components/mine/Variances/MineVarianceTable.js

```diff
@@ -83,16 +85,24 @@ export class MineVarianceTable extends Component {
       {
         title: "Submission Date",
         dataIndex: "received_date",
-        className: !this.props.isApplication ? "column-hide" : "",
-        render: (text) => <div title="Submission Date">{text}</div>,
+        className: this.hideColumn(!this.props.isApplication),
+        render: (text) => (
+          <div className={this.hideColumn(!this.props.isApplication)} title="Submission Date">
```

Comment:
Is this `className` needed? It looks like if the entire column is hidden then this cell should be automatically hidden as well

Reply:
yeah, unfortunately, the outer `className` only applies when the table is full view and hides the entire column. 

But when the table is responsive that class doesn't apply, so the same class was added to the inner div to fix the behavior when responsive. 

If the outer class is removed and only the inner div has the styling, then the header will be visible when in full view but the content will not.